### ISSUE TITLE
"Open a website/eMail" navigation behaviour no longer works

### ISSUE THREAD

**Issue by rjriel** [stefanwalther/sense-navigation#114]
### Description

When attempting to use the "Open a website/eMail" navigation behaviour a javascript error occurs and the behaviour doesn't work.

### Steps to Reproduce

1. In "Edit Mode" drag and drop the extension onto the page
2. Under "Navigation Behaviour" in the properties pane, choose "Open a website/eMail" as the "Navigation Action" and enter "http://www.google.ca" in the "Website URL"
3. Exit "Edit Mode" and click the button

##### Expected behavior

The website is opened in a new window

##### Actual behavior

Nothing. A javascript error appears in the console.

##### Operating system

```
[ ] Windows
[X] OSX
[ ] Linux
```
##### Qlik Sense

```
[ ] Desktop
[X] Enterprise
[X] Cloud
```

##### Browser

```
[X] Chrome
[ ] Firefox
[ ] Internet Explorer
[ ] Edge
[ ] Others: XXX
```

### Versions

* Extension version: v1.0.0-rc1-04
* Browser:  63.0.3239.84
* Qlik Sense: November 2017 Patch 1
* Operating system: 10.13.2
* [Other relevant versions]

**Comment by rjriel** [comment]
Issue occurs whether or not you choose to open in a new window

**Comment by rjriel** [comment]
The Error

```
TypeError: Assignment to constant variable.
    at t.$scope.doNavigate (/extensions/swr-sense-navigation/swr-sense-navigation.js?1509389618675:11)
    at t.$scope.go (/extensions/swr-sense-navigation/swr-sense-navigation.js?1509389618675:11)
    at fn (eval at compile (require.js:1), <anonymous>:4:197)
    at t (require.js:1)
    at r (require.js:1)
    at t.$eval (require.js:1)
    at t.$apply (require.js:1)
    at HTMLDivElement.<anonymous> (require.js:1)
    at HTMLDivElement.dispatch (require.js:1)
    at HTMLDivElement.g.handle (require.js:1)
```

**Comment by stefanwalther** [comment]
That's why I am starting to add tests to everything ;-)
Thx for reporting, will fix asap.

**Comment by rjriel** [comment]
Looks like something to do with the build process, the message indicates assignment of a constant and after the build this code is in the "openWebsite" switch case: `window.open((url=url)`

**Comment by stefanwalther** [comment]
Are you sure that you have deleted your cache ... I have found another problem, but actually not the one you are describing. The error you are describing actually describes something in splitToStringNum, fixing this now.

**Comment by rjriel** [comment]
Cache is deleted, still occurs. I'm not seeing `splitToStringNum` used in the `openWebsite` switch case at all though, and the `Assignment to constant variable` error message is what lead me to believe it was the `url = url` part of the code I indicated above, since the `url` variable is declared as a `const`

**Comment by stefanwalther** [comment]
👀

**Comment by stefanwalther** [comment]
Solved with dd683e1f0f6ad34eee74cd62ae1e5365257fdbd0

**Comment by stefanwalther** [comment]
What's really interesting is, that I do not see that, neither in manual nor in automated tests ...
So I could definitely not reproduce it ... Not sure why.
